pages/docs/form/switch.mdx

```diff
@@ -62,6 +62,33 @@ You can change the checked background color of the `Switch` by passing the
 </Stack>
 ```
 
+## State depending behavior
+
+States like `isDisabled` have an impact on the usability of a `Switch` and on
+the styles, except for the `isInvalid` and the `isRequired` prop.
+
+```jsx
+<FormControl as={SimpleGrid} columns={[2, null, null, 4]}>
```

Comment:
I think it's better if you used the object syntax instead of the array syntax, so you'd avoid adding `null`s.

Reply:
Yes, you are right, it is better readable. Force-pushed it.